tutors3/patches/openedx-lms-common-settings

```diff
@@ -1,6 +1,7 @@
 PROFILE_IMAGE_BACKEND = {
     "class": DEFAULT_FILE_STORAGE,
     "options": {
+        "bucket": "{{ S3_PROFILE_IMAGE_BUCKET }}",
         "location": PROFILE_IMAGE_BACKEND["options"]["location"].lstrip("/"),
         # the following non empty property is necessary in development
         "base_url": "dummyprofileimagebaseurl",
```

Comment:
Just curious, I wonder what this does? I see the comment above, but does this have any bearing on production settings?

Reply:
Hmm this followed from minio. I tested removing it and it didn't change the path of the profile image. The value of `base_url` is set in `lms.envs.common.py`: https://github.com/edx/edx-platform/blob/master/lms/envs/common.py#L4137
I'll update the PR.